src/dialogs/ha-more-info-dialog.html

```diff
@@ -33,81 +33,72 @@
           --more-info-header-background: var(--primary-color);
           --more-info-header-color: var(--text-primary-color);
         }
-        paper-dialog {
+        :host {
           max-height: 100%;
           height: 100%;
         }
       }
 
-      paper-dialog[data-domain=camera] {
+      :host([data-domain=camera]) {
         width: auto;
       }
 
-      paper-dialog[data-domain=history_graph] {
+      :host([data-domain=history_graph]) {
         width: 90%;
       }
     </style>
 
-    <!-- entry-animation='slide-up-animation' exit-animation='slide-down-animation' -->
-    <paper-dialog
-      id="dialog"
-      with-backdrop
-      opened='{{_dialogOpen}}'
-      data-domain$='[[_computeDomain(stateObj)]]'
-    >
-      <template is='dom-if' if='[[!_page]]'>
-        <more-info-controls
-          class='no-padding'
-          hass='[[hass]]'
-          state-obj='[[stateObj]]'
-          dialog-element='[[_dialogElement]]'
-          can-configure='[[_registryInfo]]'
-        ></more-info-controls>
-      </template>
-      <template is='dom-if' if='[[_equals(_page, "settings")]]'>
-        <more-info-settings
-          class='no-padding'
-          hass='[[hass]]'
-          state-obj='[[stateObj]]'
-          registry-info='{{_registryInfo}}'
-        ></more-info-settings>
-      </template>
-    </paper-dialog>
+    <template is='dom-if' if='[[!_page]]'>
+      <more-info-controls
+        class='no-padding'
+        hass='[[hass]]'
+        state-obj='[[stateObj]]'
+        dialog-element='[[_dialogElement]]'
+        can-configure='[[_registryInfo]]'
+      ></more-info-controls>
+    </template>
+    <template is='dom-if' if='[[_equals(_page, "settings")]]'>
+      <more-info-settings
+        class='no-padding'
+        hass='[[hass]]'
+        state-obj='[[stateObj]]'
+        registry-info='{{_registryInfo}}'
+      ></more-info-settings>
+    </template>
   </template>
 </dom-module>
 
 <script>
-class HaMoreInfoDialog extends window.hassMixins.EventsMixin(Polymer.Element) {
+class HaMoreInfoDialog extends window.hassMixins.DialogMixin(Polymer.Element) {
   static get is() { return 'ha-more-info-dialog'; }
   static get properties() {
     return {
-      hass: Object,
```

Comment:
You still need hass

Reply:
It's part of the mixin. 
But if that's a bit to DRY I'm happy to change that back.

Reply:
I now see you have put it in the mixin. I think it should be in the class directly, as the mixin doesn't interface with `hass` in any way.

Reply:
yes, agree. Kind of weird to have the observers on hass in the class, without the hass property tienelf. Changing.